templates/chat/request-card.hbs

```diff
@@ -10,7 +10,9 @@
                     <span class="title">{{ actor.name }}</span>
                 </span>
                 <div class="status">
-                    {{#if completed}}
+                    {{#if total includeZero=true}}
+                    <span class="total">{{ total }}</span>
```

Comment:
It would be nice to have some success/failure coloring if a target is provided when the skill is rolled.